Source/WebCore/platform/graphics/displaylists/DisplayList.h

```diff
@@ -44,6 +44,10 @@ class DisplayList {
     WTF_MAKE_NONCOPYABLE(DisplayList);
 public:
     DisplayList() = default;
+    explicit DisplayList(OptionSet<RecordReplayOption> options)
+        : m_options(options)
+    {
+    }
```

Comment:
I think if you default `options` to `{ }` you can remove the default constructor.